Hearthstone/app/src/main/java/com/example/hearthstone/adapter/HearthstoneAdapter.kt

```diff
@@ -0,0 +1,75 @@
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hearthstone.R
+import com.example.hearthstone.databinding.CardItemBinding
+
+class HearthstoneAdapter(private var cards: List<String>) :
+    RecyclerView.Adapter<HearthstoneAdapter.HearthstoneViewHolder>(){
+
+    inner class HearthstoneViewHolder(view: View): RecyclerView.ViewHolder(view) {
+        var binding = CardItemBinding.bind(view)
+
+        fun bindCard(card: String){
+            when(card){
+                "Mage" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_9)
+                }
+                "Druid" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_10)
+                }
+                "Hunter" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_11)
+                }
+                "Priest" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_12)
+                }
+                "Rogue" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_16)
+                }
+                "Paladin" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_15)
+                }
+                "Shaman" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_14)
+                }
+                "Warlock" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_13)
+                }
+                "Warrior" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_17)
+                }
+                "Demon Hunter" -> {
+                    binding.txt.text = card
+                    binding.cardImage.setImageResource(R.drawable.oval_copy_18)
+                } else ->{
+                binding.txt.text = card
+                binding.cardImage.setImageResource(R.drawable.photo_copy_2)
+                }
+            }
+        }
+    }
+    override fun onCreateViewHolder(viewGroup: ViewGroup, viewType: Int): HearthstoneViewHolder {
+        return  HearthstoneViewHolder(
+            LayoutInflater.from(viewGroup.context)
+                .inflate(R.layout.card_item, viewGroup, false)
+        )
+    }
+    override fun getItemCount(): Int = cards.size
+
+    override fun onBindViewHolder(holder: HearthstoneViewHolder, position: Int) {
+        holder.bindCard(cards[position])
+    }
+}
```

Comment:
Shouldn't see no new line
https://stackoverflow.com/questions/55086872/how-to-make-word-wrap-in-android-studio-by-default